recipes/omas/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "omas" %}
+{% set version = "0.2.6" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a3f315f4b192d665b96e4f041ce2fc7fd88a43ada9ab183249f5f5d7f54d7f6c" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+  patches:
+    - setup.py.patch
```

Comment:
It's unclear to me that this patch is required as `--single-version-externally-managed` ignores `install_requires`